lib/Completion/Bridge/TolerantParser/CompletionContext.php

```diff
@@ -367,6 +369,21 @@ public static function promotedPropertyVisibility(Node $node): bool
         return false;
     }
 
+    public static function conditionInfix(Node $node): bool
+    {
+        // If the current node is not a variable we're at the beginning of the condition like "if (<>"
+        if ($node->getText() === '') {
+            return false;
+        }
+
+        $parent = $node->getParent();
```

Comment:
It does not check (or test) the case with existing prefix (`ins` for example). Then we need to query the grandpa of `$node` :grin: